src/components/environments-summary/environment-ingress.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { faLink } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import usePollComponents from './use-poll-components';
+
+const URL_VAR_NAME = 'RADIX_PUBLIC_DOMAIN_NAME';
+const MAX_DISPLAY_NR_COMPONENT = 3;
+
+const EnvironmentIngress = ({ appName, deploymentName }) => {
+  const [componentsPollState] = usePollComponents(appName, deploymentName);
+  let components =
+    componentsPollState && componentsPollState.data
+      ? componentsPollState.data.filter(
+          component => component.variables[URL_VAR_NAME]
+        )
+      : [];
+
+  if (components.length <= 0) {
+    return <div />;
+  }
+  const tooManyPublicComponents = components.length > MAX_DISPLAY_NR_COMPONENT;
```

Comment:
`tooManyPublicComponents`? Is this a workaround for bad data?

Reply:
its a UI thing, if there is more than 3 public urls (e.g. Sindre solution), we'll only display the first 3 as it will ruin the Environment card if not